kubernetes/monitoring/scripts/victoria-metrics-restore.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+if [ $# -lt 2 ]; then
+     echo "Usage Instructions: <bucket-path> <backup service-name>"
+     echo "victoria-metrics-restore.sh cms-monitoring/vmbackup/2021/05/31/00:00 victoria-metrics"
+     echo "victoria-metrics-restore.sh cms-monitoring/vmbackup-long/2021/05/31/00:00 victoria-metrics-long"
+     exit 1;
+fi
+
+bucket_path=$1
+backup_service=$2
+
+# use tmp area to store service file
+tmpDir=/tmp/$USER/k8s/
+if [ -d $tmpDir ]; then
+    rm -rf $tmpDir
+fi
+mkdir -p $tmpDir
+cd $tmpDir
+
+curl -ksLO https://raw.githubusercontent.com/dmwm/CMSKubernetes/master/kubernetes/monitoring/services/agg/vmrestore.yaml
+#ls $tmpDir
+if [[ "$backup_service"  == "victoria-metrics" ]] ; then
+
+	echo "testing"
+  cat vmrestore.yaml | sed -e "s,cms-monitoring/vmbackup/2021/05/31/00:00,$bucket_path,g" > vmrestore.yaml.new
+  
+  curl -ksLO https://raw.githubusercontent.com/dmwm/CMSKubernetes/master/kubernetes/monitoring/services/agg/victoria-metrics.yaml
+
+  kubectl delete -f victoria-metrics.yaml
+  sleep 60s
```

Comment:
I suggest to not rely on sleep as it is unreliable condition. Any service for whatever reason may take longer. Instead, please add a loop with some interval, e..g 10s and use kubectl to check if service/deployment is gone, then if it is the case you may exit the loop. Make loop finite too to avoid infinite conditions.